docs/source/conf.py

```diff
@@ -123,13 +123,46 @@
     AttributeDocumenter,
     Documenter,
     MethodDocumenter,
+    PropertyDocumenter,
 )
 from sphinx.ext.autosummary import Autosummary  # isort:skip
 
 
+class ModinAccessorDocumenter(PropertyDocumenter):
+    """
```

Comment:
If i understand correctly, this is used for generating the doc for the class its-self like DateTime property, and the class below is used for generating documentation for the methods inside the class like dt.year etc, is that correct?

Reply:
Yes.